tests/out/glsl/variations-glsl.main.Fragment.glsl

```diff
@@ -9,7 +9,7 @@ uniform highp samplerCube _group_0_binding_0_fs;
 void main_1() {
     ivec2 sizeCube = ivec2(0);
     float a = 0.0;
-    sizeCube = textureSize(_group_0_binding_0_fs, 0).xy;
+    sizeCube = ivec2(uvec2(textureSize(_group_0_binding_0_fs, 0).xy));
```

Comment:
Because `glsl` and `spv` functions always return `sint` and, `wgsl` always returns `uints`, we get these redundant casts. Not sure if there's a way to avoid this.